### DISCUSSION TITLE
我用PCL2做我服务器的客户端启动器，有侵犯到作者吗？

### DISCUSSION THREAD

**Discussion by GmcyesG** [Meloong-Git/PCL#2157] [提问] [answered]
这是我服务器客户端
[Uploading xmsmp客户端.zip…]()

**Comment by WForst-Breeze** [comment, marked as answer]
见 #1986，只要使用的是正式版制作客户端，就都是允许的~

**Comment by PlayMcBKuwu** [comment]
> 这是我服务器客户端 Uploading xmsmp客户端.zip…

### 除非刻意：
- 使用内测版制作启动器
- 通过修改代码隐藏 PCL2 或龙腾猫跃等字样
#### 否则一般而言没什么问题
